Mage.Sets/src/mage/cards/e/EvolvedSleeper.java

```diff
@@ -0,0 +1,126 @@
+        this.addAbility(new SimpleActivatedAbility(
+                new EvolvedSleeperClericEffect(), new ManaCostsImpl<>("{1}{B}")
+        ));
+
+        // {1}{B}{B}: If Evolved Sleeper is a Phyrexian, put a +1/+1 counter on it, then you draw a card and you lose 1 life.
+        this.addAbility(new SimpleActivatedAbility(
+                new EvolvedSleeperPhyrexianEffect(), new ManaCostsImpl<>("{1}{B}{B}")
+        ));
+    }
+
+    private EvolvedSleeper(final EvolvedSleeper card) {
+        super(card);
+    }
+
+    @Override
+    public EvolvedSleeper copy() {
+        return new EvolvedSleeper(this);
+    }
+}
+
+class EvolvedSleeperClericEffect extends OneShotEffect {
+
+    EvolvedSleeperClericEffect() {
+        super(Outcome.Benefit);
+        staticText = "if {this} is a Cleric, put a deathtouch counter on it " +
+                "and it becomes a Phyrexian Human Cleric with base power and toughness 3/3";
+    }
+
+    private EvolvedSleeperClericEffect(final EvolvedSleeperClericEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public EvolvedSleeperClericEffect copy() {
+        return new EvolvedSleeperClericEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Permanent permanent = source.getSourcePermanentIfItStillExists(game);
+        if (permanent == null || !permanent.hasSubtype(SubType.CLERIC, game)) {
+            return false;
+        }
+        permanent.addCounters(CounterType.DEATHTOUCH.createInstance(), source.getControllerId(), source, game);
+        game.addEffect(new AddCardSubTypeSourceEffect(
+                Duration.Custom, SubType.PHYREXIAN, SubType.HUMAN, SubType.CLERIC
+        ), source);
+        game.addEffect(new SetPowerToughnessSourceEffect(
+                3, 3, Duration.Custom, SubLayer.SetPT_7b
+        ), source);
+        return true;
+    }
+}
+
+class EvolvedSleeperPhyrexianEffect extends OneShotEffect {
+
+    EvolvedSleeperPhyrexianEffect() {
+        super(Outcome.Benefit);
+        staticText = "if {this} is a Phyrexian, put a +1/+1 counter on it, then you draw a card and you lose 1 life";
+    }
+
+    private EvolvedSleeperPhyrexianEffect(final EvolvedSleeperPhyrexianEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public EvolvedSleeperPhyrexianEffect copy() {
+        return new EvolvedSleeperPhyrexianEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Permanent permanent = source.getSourcePermanentIfItStillExists(game);
+        if (permanent == null || !permanent.hasSubtype(SubType.PHYREXIAN, game)) {
+            return false;
+        }
+        permanent.addCounters(CounterType.P1P1.createInstance(), source.getControllerId(), source, game);
+        Player player = game.getPlayer(source.getControllerId());
+        if (player != null) {
```

Comment:
Are there instances where the player is going to be null but the creature is not?